tree/treeplayer/inc/ROOT/TDFInterface.hxx

```diff
@@ -1102,7 +1102,7 @@ protected:
             t.Fill();
          };
 
-         auto initLambda = [&t] (TTreeReader *r, unsigned int slot) {
+         auto initLambda = [&t] (TTreeReader *r, unsigned int /* unused */) {
```

Comment:
Consider still using the word 'slot' as the commented part of the parameter.  This will help a future enhancer of the semantic of the argument.